utils.py

```diff
@@ -1,8 +1,9 @@
 import os
 
+from config import black_file_pattern_list
+
 
```

Comment:
在utils.py中，get_all_files_in_folder函数被修改为生成器，这是一个很好的优化。但函数名没有反映出这一变化，可能会让调用者困惑。

```suggestion
考虑将函数名从get_all_files_in_folder改为get_all_files_in_folder_generator或者在文档字符串中明确说明它是一个生成器。